WeddingSiteBuilder/Views/profileupdate.html

```diff
@@ -6,15 +6,37 @@
     <title>Update Profile</title>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.js"></script>
     <script type="text/javascript">
+        
+            var borg1 = $('#brideorgroom1').val();
+            var fname1 = $('#firstname1').val();
+            var lname1 = $('#lastname1').val();
+            var email1 = $('#email1').val();
+            var borg2 = $('#brideorgroom2').val();
+            var fname2 = $('#firstname2').val();
+            var lname2 = $('#lastname2').val();
+            var email2 = $('#email2').val();
+
+
         var logOut = function () {
-            $.get('api/WeddingCouple/6', function (data) { console.log(data); });
+            $.get('api/logout');
```

Comment:
we're probably going to need a callback for every api call. Just make one and put some comments on it.